diagnostic.md

```diff
@@ -57,6 +67,8 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
+$('find("li")'),
```

Comment:
We need to select a `<li>` element and invoke methods to traverse to `<body>` and then hide it.  One possible way is `$('li').parent().parent().hide();`.